packages/esm-home-app/src/index.ts

```diff
@@ -24,6 +24,8 @@ export const homeWidgetDbLink = getSyncLifecycle(createDashboardLink(dashboardMe
 
 export const homeWidgetDashboard = getSyncLifecycle(homeWidgetDashboardComponent, options);
 
+export const metrics = getAsyncLifecycle(() => import('./metrics/metrics.component'), options);
```

Comment:
```suggestion
export const metrics = getSyncLifecycle(Metrics, options);
```

Reply:
Add the import `Metrics` from `./metrics/metrics.component` at the top of the file

Reply:
Isn't `getAsyncLifecycle` better for app performance? Or does it have specific use cases?

Reply:
I think there's a bit of misalignment on when to use `getAsyncLifecycle` and when to use `getSyncLifecycle` 

`homeWidgetDashboard` uses `getSyncLifecycle` to load a component that is similar to metrics. Let's leave as is for now and will circle back.